packages/dx-react-scheduler/docs/reference/all-day-panel.md

```diff
@@ -45,6 +45,8 @@ Field | Type | Description
 ------|------|------------
 startDate | Date | The cell's start time.
 endDate | Date | The cell's end time.
+groupingInfo? | Array&lt;[Group](./grouping-panel.md/#group)&gt; | Information about the cell's group(s).
```

Comment:
```suggestion
groupingInfo? | Array&lt;[Group](./grouping-panel.md/#group)&gt; | Information about the cell's grouping.
```

Reply:
Please make the same change for every `groupingInfo` description that involves cell.